lib/plugins/aws/deploy/compile/events/apiGateway/lib/cors.test.js

```diff
@@ -55,16 +55,19 @@ describe('#compileCors()', () => {
         origins: ['*'],
         headers: ['*'],
         methods: ['OPTIONS', 'PUT'],
+        allowCredentials: false,
```

Comment:
The code in validate.js applies the default so there is no default logic to test here. `compileCors` operates on the validated input (see [cors.js](https://github.com/asprouse/serverless/blob/d5b64c8c7113da9429655a409f3ef48a6827dee6/lib/plugins/aws/deploy/compile/events/apiGateway/lib/cors.js#L9)). I am just following the convention established in the existing code.